aou/aou-rstudio-bioconductor/rstudio/scripts/init_userconf.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/with-contenv bash
+# shellcheck shell=bash
+
+## Set defaults for environmental variables in case they are undefined
+DEFAULT_USER=${DEFAULT_USER:-rstudio}
+USER=${USER:=${DEFAULT_USER}}
+USERID=${USERID:=1000}
+GROUPID=${GROUPID:=1000}
```

Comment:
These are not user-overridable right?

Reply:
Correct, RStudio remaps/regroups UID and GID at runtime inside `/init`